index.html

```diff
@@ -192,7 +192,7 @@ <h5>
                   <div class="col-md-11 pivx-feature-interior">
 
                     <div class="col-md-12 feature-icon">
-                      <img src="https://pivx.org/build/images/content/img_privacy.png" alt="PIVX Privacy" class="smaller-feature">
+                      <img src="https://pivx.org/build/images/content/img_privacy.png" alt="P1IVX Privacy" class="smaller-feature">
```

Comment:
nit: this was possibly an accident?

Reply:
> The majority of the code is great! Some minor nits left, and I will do any style cleanup post-merge.
> 
> There is one **major** thing I've realised during review + after running some tests: MPW cipherkeys can be recognised with the below practice:
> 
> * We know WIF keys are fixed-length.
> * We know the PBKDF2 password key is fixed-length.
> * We know AES output natively contain non-Base58 chars (i.e: `/, 0, O, I, l`).
> 
> I conclude that our cipherkeys are **always** a fixed 1024 bits, or 128 bytes (`.length`), and will always include non-Base58 without the manual addition of a `+` symbol.
> 
> If you change the implementation to check for an input length of `128`, and check for non-Base58 using [my existing code](https://github.com/PIVX-Labs/MyPIVXWallet/blob/1d37032a91fda2f4f8050ea0158493592af01cb7/index.html#L875), we can have a very clean and safe implementation, and then we'll merge! rocket

Ok I agree that the + can be removed and looking at the length is better (which will fixed and different for plain and ciphered). However I think that it is possible (even if very unlikely) that the ciphered message in base64 doesn't contain any non-base58 characters.
So I will just check for length, what you think?

Reply:
That should also work fine! Since it should be impossible for a PIVX WIF and an MPW cipherWIF to be the same length, actually. :+1: